tests/Unit/LocationCoordinatesTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+use LibreNMS\Tests\TestCase;
+
+class LocationCoordinatesTest extends TestCase
+{
+    /**
+     * Regexes
+     */
+    private array $regexes = [
```

Comment:
Close, but you need to test the actual code in case someone changes the regex. I think this is how that would look:
```php
$location = new Location(['location' => 'test data']);
$location->parseCoordinates();
$this->assertEquals(99.999, $location->lat);
$this->assertEquals(99.999, $location->lng);
```